internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -350,24 +386,24 @@ func (e *BootstrapEnvStackConfig) Parameters() ([]*cloudformation.Parameter, err
 
 // SerializedParameters returns the CloudFormation stack's parameters serialized
 // to a YAML document annotated with comments for readability to users.
-func (e *BootstrapEnvStackConfig) SerializedParameters() (string, error) {
+func (e *BootstrapEnv) SerializedParameters() (string, error) {
 	// No-op for now.
 	return "", nil
 }
 
 // Tags returns the tags that should be applied to the bootstrap CloudFormation stack.
-func (e *BootstrapEnvStackConfig) Tags() []*cloudformation.Tag {
-	return (*EnvStackConfig)(e).Tags()
+func (e *BootstrapEnv) Tags() []*cloudformation.Tag {
+	return (*Env)(e).Tags()
 }
 
 // StackName returns the name of the CloudFormation stack (based on the app and env names).
-func (e *BootstrapEnvStackConfig) StackName() string {
-	return (*EnvStackConfig)(e).StackName()
+func (e *BootstrapEnv) StackName() string {
+	return (*Env)(e).StackName()
 }
 
 // ToEnv inspects an environment cloudformation stack and constructs an environment
 // struct out of it (including resources like ECR Repo).
-func (e *BootstrapEnvStackConfig) ToEnv(stack *cloudformation.Stack) (*config.Environment, error) {
+func (e *BootstrapEnv) ToEnv(stack *cloudformation.Stack) (*config.Environment, error) {
```

Comment:
nit: this method name is slightly off now that we have separate `stack.Env` and `stack.EnvConfig` 😂  types maybe `ToEnvMetadata`?